src/redux/features/categories/categoriesSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  currentStatus: 'Under Construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => state.currentStatus,
+  },
+});
```

Comment:
- [ ] Initial state is expected to have categories state to be empty, just as you did, and that state is what you have to check its value if it has the text "Under Construction" and return that else return the state value. 
What if you could implement it this way below?

```javascript
reducers: {
    checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
  },
```

Reply:
@AmaduKamara, thanks for the review and suggestions. but the requirement says that the function should always to return "under construction"
![image](https://user-images.githubusercontent.com/51995150/229762961-4c7ef6df-b26e-4809-84c7-764e5696b5b0.png)

Reply:
Thanks again @AmaduKamara, I will try your method now.